spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -134,9 +134,9 @@
           }
         end
       end
-      it "only try to load a whitelist once" do
+      it 'only try to load a whitelist once' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.